zds/api/bits.py

```diff
@@ -28,3 +31,18 @@ def get_data(self, **kwargs):
             kwargs['params'].append(rqp_pv)
 
         return super(DJRF3xPaginationKeyBit, self).get_data(**kwargs)
+
+
+class UpdatedAtKeyBit(KeyBitBase):
+    update_key = None
+
+    def __init__(self, update_key, params=None):
+        super(UpdatedAtKeyBit, self).__init__(params)
+        self.update_key = update_key
+
+    def get_data(self, **kwargs):
+        value = cache.get(self.update_key, None)
+        if not value:
```

Comment:
Non non, c'est pas juste une question de `not`. Si la valeur de ton cache est `0`, tu vas entrer dans ton `if`, parce que `not 0 == True`.

`cache.get` retourne `None` s'il ne trouve pas la clé, tu dois regarder si `value` est `None`, pas si `value` est falsy.